test/src/app.js

```diff
@@ -1,9 +1,179 @@
+import React from 'react';
+import IField, { CARD_TYPE, CVV_TYPE, CardknoxApplePay } from '@cardknox/react-ifields';
 
-import AppForm from './Components/form';
+export default class App extends React.Component {
+    constructor(props) {
+        super(props);
+        const account = {
+            xKey: "",
+            xSoftwareName: "react-cardknox-ifields",
+            xSoftwareVersion: "1.0.0"
+        };
+        const threeds = {
+            enable3DS: false,
+            waitForResponse: false,
+            waitForResponseTimeout: undefined,
+            amount: 0,
+            month: "01",
+            year: "2020"
+        };
+        const ccoptions = {
+            placeholder: 'Card Number',
+            enableLogging: true,
+            autoFormat: true,
+            autoFormatSeparator: ' ',
+            autoSubmit: false,
+            iFieldstyle: {
 
-export default function App() {
+            },
+            iFrameStyle: {
 
-    return (
-        <AppForm />
```

Comment:
Can the apple pay component be added to the `form` component?
Removing that component removes all the 3DS sample code

Reply:
Done